src/main/java/com/sprint/mission/discodeit/controller/ReadStatusController.java

```diff
@@ -0,0 +1,60 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.docs.ReadStatusSwagger;
+import com.sprint.mission.discodeit.dto.request.ReadStatusCreateRequest;
+import com.sprint.mission.discodeit.dto.request.ReadStatusUpdateRequest;
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.service.ReadStatusService;
+import java.util.List;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/read-status")
+@RequiredArgsConstructor
+public class ReadStatusController implements ReadStatusSwagger {
+
+  private final ReadStatusService readStatusService;
+
+  @RequestMapping(method = RequestMethod.POST,
+      consumes = MediaType.APPLICATION_JSON_VALUE
+  )
```

Comment:
`@RequestMapping` 대신 각 HTTP 메서드마다 맞는 애노테이션을 적용해도 좋을 것 같네요.  
```suggestion
  @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
```